pkg/leader/leader.go

```diff
@@ -43,16 +51,37 @@ func run(ctx context.Context, namespace, name string, client kubernetes.Interfac
 		logrus.Fatalf("error creating leader lock for %s: %v", name, err)
 	}
 
-	t := time.Second
-	if dl := os.Getenv("CATTLE_DEV_MODE"); dl != "" {
-		t = time.Hour
+	leaseDuration := defaultLeaseDuration
+	renewDeadline := defaultRenewDeadline
+	retryPeriod := defaultRetryPeriod
+	if d := os.Getenv("CATTLE_DEV_MODE"); d != "" {
+		leaseDuration = developmentLeaseDuration
+		renewDeadline = developmentRenewDeadline
+	}
+	if d := os.Getenv("CATTLE_ELECTION_LEASE_DURATION"); d != "" {
+		leaseDuration, err = time.ParseDuration(d)
+		if err != nil {
+			return errors.Wrapf(err, "CATTLE_ELECTION_LEASE_DURATION is not a valid duration")
+		}
+	}
+	if d := os.Getenv("CATTLE_ELECTION_RENEW_DEADLINE"); d != "" {
+		renewDeadline, err = time.ParseDuration(d)
+		if err != nil {
+			return errors.Wrapf(err, "CATTLE_ELECTION_RENEW_DEADLINE is not a valid duration")
+		}
+	}
+	if d := os.Getenv("CATTLE_ELECTION_RETRY_PERIOD"); d != "" {
+		retryPeriod, err = time.ParseDuration(d)
+		if err != nil {
+			return errors.Wrapf(err, "CATTLE_ELECTION_RETRY_PERIOD is not a valid duration")
```

Comment:
Nit: could include the value in these error messages
```suggestion
			return errors.Wrapf(err, "CATTLE_ELECTION_RETRY_PERIOD value [%s] is not a valid duration", d)
```

Reply:
addressed as https://github.com/rancher/wrangler/commit/57d388e630d8c3d1726e7ef7e8bf1066eb12a930